python/cudf/cudf/pylibcudf_tests/test_json.py

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+import io
+
+import pandas as pd
+import pyarrow as pa
+import pytest
+from utils import sink_to_str
+
+import cudf._lib.pylibcudf as plc
+
+
+@pytest.mark.parametrize("rows_per_chunk", [8, 100])
+@pytest.mark.parametrize("lines", [True, False])
+def test_write_json_basic(table_data, source_or_sink, lines, rows_per_chunk):
+    plc_table_w_meta, pa_table = table_data
+    sink = source_or_sink
+
+    kwargs = dict()
+    if rows_per_chunk <= plc_table_w_meta.tbl.num_rows():
+        kwargs["rows_per_chunk"] = rows_per_chunk
```

Comment:
Do we need this condition because if you pass `rows_per_chunk > num_rows` libcudf spits an error?

Reply:
Or are you trying to test with default `rows_per_chunk=None`, if the latter, please add `None` as a parametrization.

Reply:
> Do we need this condition because if you pass `rows_per_chunk > num_rows` libcudf spits an error?

Yes.

Reply:
Perhaps that is a feature request, because that seems bad, I need to know the maximum number of rows in my file before I set rows_per_chunk.